osdep/subprocess.c

```diff
@@ -45,6 +45,13 @@ static void *run_subprocess(void *ptr)
     return NULL;
 }
 
+void mp_devnull(void *ctx, char *data, size_t size)
+{
+    (void)ctx;
+    (void)data;
+    (void)size;
```

Comment:
I don't think we enable warnings for unused parameters? We don't try to silence them anywhere either.